consensus/istanbul/backend/engine.go

```diff
@@ -1017,6 +1026,9 @@ func (sb *Backend) SetStartValidatingBlock(blockNumber *big.Int) error {
 
 // SetStopValidatingBlock sets the block that the validator will stop just before (exclusive range)
 func (sb *Backend) SetStopValidatingBlock(blockNumber *big.Int) error {
+	if sb.replicaState == nil {
+		return errors.New("Not configured as a validator")
```

Comment:
I would create an error variable with this error and resuse it. `NotAValidatorError` or something